src/sharedData/visualization/components/VisualizationConfigForm/SetDatasetStep.js

```diff
@@ -300,6 +299,7 @@ const DataColumns = props => {
         aria-labelledby="dataset-config-dataColumns-label"
         value={option}
         onChange={event => onChange({ ...value, option: event.target.value })}
+        error={!!fieldState?.error || undefined}
```

Comment:
question: what are the possible value for fieldState.error? Would undefined not be the fallback value anyway?